karma-shared.conf.js

```diff
@@ -65,13 +65,13 @@ module.exports = function(config, specificOptions) {
       'SL_Safari-1': {
         base: 'SauceLabs',
         browserName: 'safari',
-        platform: 'OS X 10.12',
+        platform: 'OS X 10.13',
         version: 'latest-1'
       },
       'SL_Safari': {
         base: 'SauceLabs',
         browserName: 'safari',
-        platform: 'OS X 10.12',
+        platform: 'OS X 10.13',
```

Comment:
This may start failing soon as Safari 12 might only be available in Mojave (that's how it is in BrowserStack, for example) - i.e. `10.14`. Do we have to specify the platform? If the answer's yes, maybe Safari cannot really be used as `latest` & `latest-1` versions...

Reply:
The same remark applies to iOS BTW.

Reply:
When it starts to fail, we can update it. But I can also try it without the explicit platform.